minion/frontend/utils.py

```diff
@@ -11,7 +11,19 @@
 DEFAULT_FRONTEND_CONFIG = {
     'backend-api': {
         'url': 'http://127.0.0.1:8383'
+    },
+    'login_conf': {
+       'login_type': 'persona', # persona OR ldap
+
+       'ldap_uri': 'ldaps://ldap.server/',
```

Comment:
The ldap things should all be part of their own associative array; login['ldap']['uri'] and so on.